src/main/java/com/falkordb/impl/resultset/ResultSetScalarTypes.java

```diff
@@ -14,7 +14,8 @@ enum ResultSetScalarTypes {
     VALUE_NODE,
     VALUE_PATH,
     VALUE_MAP,
-    VALUE_POINT;
+    VALUE_POINT,
+    VALUE_VECTORF32;
```

Comment:
not familiar with Java, but can you explicitly assign values to enums ? and not relay on ordering of the values?

Reply:
there is no built in way like in C to do it, you can define a constructor and a field  

```java
public enum ResultSetScalarTypes {
    VALUE_NULL(0),
    VALUE_STRING(1),
    VALUE_INTEGER(2);

    private final int code;

    // Constructor
    ResultSetScalarTypes(int code) {
        this.code = code;
    }

    // Getter for the code
    public int getCode() {
        return code;
    }
}
```